src/ztc/datamodel/tests/base_tests.py

```diff
@@ -2,12 +2,13 @@
 
 from django.utils import timezone
 
-from zds_schema.constants import RolOmschrijving
-
-from ztc.datamodel.choices import (
-    JaNee, ObjectTypen, VertrouwelijkheidAanduiding
+from zds_schema.constants import (
+    RolOmschrijving,
+    VertrouwelijkheidsAanduiding as VertrouwelijkheidAanduiding
```

Comment:
probably better to not define an alias, but update the references to the `zds_schema` import